### PR TITLE
Add signup option

### LINKED ISSUE #230: Add option to signup from the chromebot
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
Currently no option for signup is there

**Expected behavior:**
Add option of signup through signup

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by ms10398**
Fixes #230 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Added option of signup from the chromebot

### Screenshot
screenshot from 2018-01-11 10-24-59
screenshot from 2018-01-11 10-24-19

### CODE REVIEW

File: src/signup.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                            <br />
+                            <div class="form-group">
+                                <label for="password">Password:</label>
+                                <input type="password" class="form-control" id="password" placeholder="Enter your password">
+                            </div>
+                            <br />
+                            <button type="submit" class="btn btn-success" id="signupbutton">Signup</button>
+                            <br>
+                            <p style="text-align: center;">
+                                <a href="login.html" style="color: #fff; ">Already Registered?Want to Login</a>
```
Comment (stealthanthrax) [inline src/signup.html]: Could you add a space between "?" or add a br tage

Comment (ms10398) [reply src/signup.html]: Check it now @stealthanthrax

**Comment by ms10398** [comment]
Check out now @stealthanthrax @0x48piraj @sachincool @gabru-md

File: src/js/signup.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	$.ajax({
+		url: signupEndPoint,
+		dataType: "jsonp",
+		jsonpCallback: "p",
+		jsonp: "callback",
+		crossDomain: true,
+		success: function (response) {
+			if(response.accepted){
+				alert(response.message);
+				console.log(response);
```
Comment (gabru-md) [inline src/js/signup.js]: Remove console log

**Comment by ms10398** [comment]
@gabru-md Have a look again

**Pull request merged**